### DISCUSSION TITLE
Error in installing iso image in Vmware

### DISCUSSION THREAD

**Discussion by nehahussain** [Security-Onion-Solutions/securityonion#8080] [Unsupported Versions]
it is giving me error in installation in the last step.
"Install had a problem. please see /root/sosetup.log for details.
A summary of errors can be found in /root/errors.log."
![WhatsApp Image 2022-06-06 at 10 44 26 PM](https://user-images.githubusercontent.com/56157450/172216828-177a17b6-9296-4c9b-b896-aa3c129e0973.jpeg)

I checked the errors.log file and it shows me
![erro](https://user-images.githubusercontent.com/56157450/172216844-ea4fd76e-633c-4249-905f-84bcaf27f659.PNG)

Please if someone can help me , I have my submission in few hours , I have to perform lab in security onion

**Comment by nehahussain** [comment]
Please someone help me, I am facing this issue for 10 days. 
Please can someone help me install this?

**Comment by dougburks** [comment]
What version of Security Onion are you trying to install?

Are you using our Security Onion ISO image?  If so, did you verify the ISO image as shown at https://securityonion.net/download?

What are the specs of the VM? Do they meet the minimum requirements shown at https://docs.securityonion.net/en/2.3/hardware.html?

What options did you choose during Setup?

Have you tried doing a simple Import installation as shown at https://docs.securityonion.net/en/2.3/first-time-users.html?

Are you able to share the full errors.log and sosetup.log (not just a screenshot)?

**Comment by nehahussain** [comment]
Do I need to verify? Because I download from your GitHub repo, which you
mentioned in my previous question that it’s not corrupt it’s fine.

**Reply by dougburks** [reply]
Yes, you need to verify your download. In my response to your previous question, I verified that the file itself is not corrupt and that I was able to download it without any interruption. If your download is being interrupted or interfered with, then you need to try again until you get a good copy and are able to verify it successfully.

From https://docs.securityonion.net/en/2.3/download.html:
![image](https://user-images.githubusercontent.com/1659467/172598711-a9b4e046-0ff3-47b2-a10a-271cce7af2e3.png)

**Comment by nehahussain** [comment]
From where I can get the commands to verify the iso image in windows host
machine. ?
Since my host machine is windows and the commands To verify in GitHub are
of Ubuntu.

**Reply by dougburks** [reply]
You can verify the ISO image using the MD5, SHA1, or SHA256 hashes or the GPG signature shown at https://securityonion.net/download. There are many different ways to do this kind of verification on Windows. Try Googling something like `windows verify md5`.
